examples/doc/pyomobook/pyomo-components-ch/param_misc.txt

```diff
@@ -43,6 +43,6 @@
 1 Objective Declarations
     o : Size=1, Index=None, Active=True
         Key  : Active : Sense    : Expression
-        None :   True : minimize : p[1] * x[1] + p[2] * x[2] + p[3] * x[3]
+        None :   True : minimize : x[1] + p[2]*x[2] + p[3]*x[3]
```

Comment:
This is a bug (a bad baseline update)!  p[1] is a *mutable* Param and should not be eliminated from the expression.

Reply:
The baseline wasn't up-to-date.

Reply:
I don't understand.  That example creates 2 models and pprints them both.  In the first model, 
```
model.p = Param(model.A, initialize=p)
```
`model.p` is immutable, so the `p`'s should (and ARE) removed from the expression.  That pprint ends at line 22 in the baseline.  In the second model, 
```
model.p = Param(model.A, initialize=p, mutable=True)
```
`model.p` is *mutable*.  Therefore,
```
model.o = Objective(expr=summation(model.p, model.x))
```
Should generate an expression `p[1]*x[1] + p[2]*x[2] + p[3]*x[3]` and **_NOT_** `x[1] + p[2]*x[2] + p[3]*x[3]`.  This baseline is *wrong*.